scripts/v2/package-helm-manifest.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script generates helm manifest and replaces required values in helm chart.
+
+set -e
+
+LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE=$1
+PUBLIC_REGISTRY=$2
+VERSION=$3
+DIR=$4
```

Comment:
Do a proper parsing loop to handle commandline arguments and provide a usage message if the user gets it wrong.

Examples
* [clean-orphan-role-assignments.sh](https://github.com/Azure/azure-service-operator/blob/main/scripts/v2/clean-orphan-role-assignments.sh#L23)
* [create-kind-wi-storage.sh](https://github.com/Azure/azure-service-operator/blob/main/scripts/v2/create-kind-wi-storage.sh#L18)
* [delete-kind-wi-storage.sh](https://github.com/Azure/azure-service-operator/blob/main/scripts/v2/delete-kind-wi-storage.sh#L17)